docker/nginx/nginx.conf

```diff
@@ -0,0 +1,88 @@
+#ssl_certificate /etc/letsencrypt/live/myopenhab.org/fullchain.pem;
+#ssl_certificate_key /etc/letsencrypt/live/myopenhab.org/privkey.pem;
+worker_processes 4;
+
+events { worker_connections 1024; }
+
+http {
+
+upstream app_server {
+        server app-1:3000 fail_timeout=5s max_fails=5;
+}
+
+server {
+    listen 8080 default_server;
+    listen [::]:8080 default_server;
+    #server_name localhost;
+    #return 301 https://$server_name$request_uri;
+
+
+    charset utf-8;
+
+    access_log /var/log/nginx/openhab.org-access.log;
+    error_log /var/log/nginx/openhab.org-error.log;
```

Comment:
Will commit this after documenting the current state. But as mentioned cool idea.